### PR TITLE
update FAQ section 3.13

### PR DESCRIPTION
**Pull request by 4current**
Some minor changes to correct and clarify FAQ section 3.13.

### CODE REVIEW

File: docs/FAQ.rst [hunk 1]
```diff
@@ -1023,17 +1023,17 @@ to your ~/GNUstep/Defaults/Windowmaker file.
 How do I get the classic NeXT(tm)-like style back?
 ..................................................
 
-Open WPrefs and go under the ``Other Configurations`` tab. Under ``Title Bar
-Style``, select the classic look.
+Open WPrefs and navigate to the ``Other Configurations`` tab with the scroll bar. Within the ``Titlebar
```
Comment (gryf) [inline docs/FAQ.rst]: Lines should not exceed 80 characters, just to be coherent with rest of the file.

**Review by gryf — CHANGES_REQUESTED** [review]
Thanks for the contribution! I have one small remark as a inline comment.

File: docs/FAQ.rst (hunk 1 above)
Comment (4current) [reply docs/FAQ.rst]: Thanks, I'll make the change.

**Review by gryf — APPROVED** [review]
Great, thanks!

**Pull request merged**